config/config/sc-config.yaml

```diff
@@ -46,8 +46,6 @@ user:
   alertmanager:
     group_by:
       - cluster
```

Comment:
Should this one still be there?

Reply:
I don't think so. My guess is that it would group everything from one cluster into one giant alert (though I might be wrong)

Reply:
Will see if we can test this in our long-running cluster!